webpack.config.js

```diff
@@ -1,33 +1,13 @@
-var webpack = require('webpack');
-
-var dedup =
-  new webpack.optimize.DedupePlugin();
-
 module.exports = {
   entry: {
     app: ['./src/index.js']
   },
   module: {
     loaders: [
-      { test: /\.js$/, exclude: [/node_modules/], loader: 'babel-loader' }, //'react-hot',
-      { test: /\.scss$/, loader: "style-loader!css-loader!sass-loader"},
-      { test: /\.styl$/, loader: 'style-loader!css-loader!stylus-loader?paths=node_modules/'},
-      { test: /\.less$/, loader: "style-loader!css-loader!less-loader" },
-      { test: /\.(svg|png|gif)$/, loader: "url-loader?limit=8192" }
+      {test: /\.js$/, exclude: [/dist/, /node_modules/], loader: 'babel'}
```

Comment:
This is the only loader we actually used. No CSS, SASS, Stylus, or image files are included in this repo.